src/main.rs

```diff
@@ -30,7 +31,7 @@ fn main() -> Result<()> {
         }
     }
 
-    let config = Config::new(current_dir().unwrap().join("resources"));
+    let config = Config::new(Config::resources_dir_path().unwrap());
```

Comment:
I prefer the path being done in `main.rs`. So whole lib doesn't depend on `cargo_packager_resource_resolver`. Also I think we don't need a packager. Just need to get the directory from `cargo_packager_resource_resolver` directly.